### PR TITLE
layout recommend product

### PR DESCRIPTION
**Pull request by at-trangle**
## Description
layout recommend product
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/public/pages/detail.blade.php [hunk 1]
```diff
@@ -69,6 +70,168 @@
...
+                          </a>
+                        </div>
+                      </div>
+                    </figure>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        <!-- <div class="clearfix"> </div> -->
```
Comment (at-phuongvu) [inline resources/views/public/pages/detail.blade.php]: `comment` line nè

**Comment by at-phuongvu** [comment]
LGTM

File: public/js/public/details.js [hunk 2]
```diff
@@ -44,3 +44,41 @@ function loadPage() {
 }
 
 loadPage();
+function recommendProduct() {
+    $('.list-recomment-product').each(function(){
+        if ( $().owlCarousel ) {
```
Comment (at-quangtran) [inline public/js/public/details.js]: $().owlCarousel là gì e?

Comment (at-trangle) [reply public/js/public/details.js]: cái này e cx lấy của team FE thôi a...chắc là $(this).owlCarousel ạ

**Pull request merged**